### ISSUE TITLE
Images that are submitted are loaded via http instead of https

### ISSUE THREAD

**Issue by poVoq** [QuantumObject/docker-ushahidi-platform#15]
When you upload an image to a survey and view it in the web-client is it loaded from the server without encryption (http) which causes an ugly browser warning.

Not sure if this is a problem with this docker image or an upstream issue.

Thanks a lot for making this work!

**Comment by angelrr7702** [comment]
You are welcome ... I used two other container for proxy and https with let encrypts .  for that reason I don't provide https at container but at the same time each open source its is different complication when dealing with https . Firefox and chrome too consider some element insecure and sometimes stop script from working .. normal is a problem of permission . 

thttps://github.com/jwilder/nginx-proxy
https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion

**Comment by angelrr7702** [comment]
change permission at /var/www/platform/storage only www-data user  (apache2,PHP) have right to modified it .

**Comment by poVoq** [comment]
I updated to your latest docker image, which seems to address the permission issue, but the mixed content warning remains. The error message in Firefox is this (sorry in German):
`Laden von gemischten aktiven Inhalten "http://api.cxb.reports.watsan.eu/api/v3/collections?orderby=created&order=DESC" wurde blockiert.`
It happens as soon as you open the popup on a map-marker that includes an user-uploaded image.

**Comment by angelrr7702** [comment]
link create internal by the platform is the form http:// when suppose to be on the form https:// when using https  and that create the mixed-content website that fail to load in the browner correctly.

**Comment by poVoq** [comment]
If I add that right after the RewriteEngine On
I get the `Sorry, something went wrong. Try reloading the page.` error

**Comment by angelrr7702** [comment]
https://help.dreamhost.com/hc/en-us/articles/215747758-Force-your-site-to-load-securely-with-an-htaccess-file

**Comment by angelrr7702** [comment]
if something go wrong try to modified file back to original and try again ...

**Comment by poVoq** [comment]
Returning to the original works of course.

**Comment by angelrr7702** [comment]
checking , when you create the client container did you defined  PLATFORM with https://ip_platform ?

 docker run -d -p 80 -e PLATFORM="http://ip_platform:port" quantumobject/docker-ushahidi-client

**Comment by poVoq** [comment]
> checking , when you create the client container did you defined PLATFORM with https://ip_platform ?

Yes, however with the actual URL not an IP. SSL via let's encrypt.

**Comment by angelrr7702** [comment]
update , need to check if problem still present
